src/ui/ui_render/ui_render_mixin.js

```diff
@@ -54,6 +55,26 @@ export function uiRenderMixin(kbnServer, server, config) {
   // expose built css
   server.exposeStaticDir('/built_assets/css/{path*}', fromRoot('built_assets/css'));
 
+  server.route({
+    path: '/bundles/translations/{locale}.json',
```

Comment:
__note:__ if though we support one language at time, we want to have a different HTTP cache entry for every language.